README.rst

```diff
@@ -1,12 +1,47 @@
-Orange Network
-==============
+Orange3 Network
+===============
 
-Orange Network is an add-on for Orange_ data mining software package. It
-provides network visualization and analysis tools.
+Orange3-Network is an add-on for Orange_ data mining software package. It
+provides network visualization and network analysis tools.
 
 .. _Orange: http://orange.biolab.si/
 
 Documentation is found at:
 
-http://orange-network.readthedocs.org/
+http://orange3-network.rtfd.io
 
+Installation
+------------
+
+Install from Orange add-on installer through Options - Add-ons.
+
+To install the add-on with pip use
+
+    pip install orange3-network
+
+To install the add-on from source, run
+
+    python setup.py install
+
+To register this add-on with Orange, but keep the code in the development directory (do not copy it to 
+Python's site-packages directory), run
+
+    python setup.py develop
+
+Windows users have to compile the code. Run
+
+    python setup.py build_ext -i --compiler=msvc install
```

Comment:
If the compiler is MinGW, the flag is still `--compiler msvc`? If not, I suggest setting it to mingw as that's lighter and easier to install. And mentioning that a compiler is needed seems reasonable. Even on GNU/Linux, the user needs to have a compiler package installed (which is, albeit with `apt-get install g++` much easier, usually not the default case).